.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,52 @@
+name: Build docs
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.7
```

Comment:
Could you bump this python version up to 3.10 or 3.11? 3.7 has reached end of life, so it's better if we don't use it